docs/framework/docker/aspnetmvc.md

```diff
@@ -0,0 +1,240 @@
+build the assets to publish using Visual Studio. You'll use the Docker tools
+to build the image that contains your application, and
+runs that application when it is started. You'll see how to connect to your
+application running in a Windows based Docker container.
+
+The application you'll run in a container is a simple website that
+answers questions randomly. This application
+is a basic MVC application with no authentication support, or database
+storage. That lets you focus on moving the web tier to a container. Later
+topics show how to move and manage persistent storage in containerized
+applications.
+
+The finished application is located in the
+[dotnet/core-docs repository on GitHub](https://github.com/dotnet/core-docs/tree/master/samples/framework/docker/MVCRandomAnswerGenerator).
+
+You need to be familiar with some Docker terms before you begin working
+on moving your application to a container.
+
+> A **Docker image** is a read-only template that defines the environment
+> for a running container, including the OS, system components, and application(s).
+
+One important feature of Docker images is that images are composed from a
+base image. Each new image adds a small set of features to an existing
+image. 
+
+> A **Docker container** is a running instance of an image. 
+
+You scale an application by running the same image in many containers.
+Conceptually, this is similar to running the same application in multiple
+hosts.
+
+You can learn more about the Docker architecture by reading the 
+[Docker Overview](https://docs.docker.com/engine/understanding-docker/)
+on the Docker site. 
+
+Before starting, you need to install [Docker for Windows](https://docs.docker.com/docker-for-windows/).
+You need to install version 1.12 Beta 26, or newer for Windows container support.
+After installing and starting Docker, you'll need to right-click on the
+tray icon and select 'Switch to Windows containers...' in order to run
+Docker images based on Windows OSs. This command takes a few seconds to
+execute.
+
+Moving your application involves these steps:
+
+1. Creating a publish task to build the assets for an image.
+2. Building a Docker image that will run your application.
+3. Starting a Docker container that runs your image.
+4. Verifying the application using your browser.
+
+## Publish script
+
+You can use the Visual Studio Publish command to create a publish profile
+for your application. You'll copy all the published assets to your Docker
+image later in this tutorial. The publish command will put all the publish
+assets in one directory tree that you will copy to your target image.
+Right-click on your solution in Visual Studio, and select "Publish".
+Click the "Custom" profile button, and then select "File System" as the
+method (see image). Choose a new directory in your solution folder. (The
+downloaded sample uses "containerImage" as the folder name).
+
+![Publish Connection][publish-connection]
```

Comment:
should your screenshot show more of the path? for example, let's says that instead of C:\users\wiwagn\documents\GitHub, the path starts at C:\GitHub\ for example.